client/pov-recovery/src/lib.rs

```diff
@@ -175,51 +182,45 @@ where
 		}
 
 		let hash = header.hash();
+
+		if self.pending_candidates.contains_key(&hash) {
```

Comment:
Pending candidates should be renamed, as these are now not anymore "pending candidates"